src/js/api/config.js

```diff
@@ -19,7 +19,23 @@ define([
         mute: false,
         volume: 90,
         width: 480,
-        height: 270
+        height: 270,
+        aria: {
+            'jw-icon-play': 'Play',
+            'jw-icon-playback': 'Start playback',
+            'jw-icon-pause': 'Pause',
+            'jw-icon-volume': 'Volume',
+            'jw-icon-prev': 'Previous',
+            'jw-icon-next': 'Next',
+            'jw-icon-cast': 'Chromecast',
+            'jw-icon-fullscreen': 'Fullscreen',
+            'jw-icon-playlist': 'Playlist',
+            'jw-icon-hd': 'Quality',
+            'jw-icon-cc': 'Closed captions',
+            'jw-icon-audio-tracks': 'Audio tracks',
+            'jw-icon-replay': 'Replay',
+            'jw-icon-buffer': 'Loading'
+        }
```

Comment:
Let's use simpler names that don't match the CSS classes and allow for dot syntax. This `aria` block should look like:
```js
localization: {
  play: 'Play',
  playback: 'Start playback',
  pause: 'Pause',
  volume: 'Volume',
  prev: 'Previous',
  next: 'Next',
  cast: 'Chromecast',
  fullscreen: 'Fullscreen',
  playlist: 'Playlist',
  hd: 'Quality',
  cc: 'Closed captions',
  audioTracks: 'Audio tracks', // need to convert camelCase to hyphens
  replay: 'Replay',
  buffer: 'Loading'
}
```